templates/_partials/microdata-jsonld.tpl

```diff
@@ -96,10 +96,19 @@
     "description": "{$page.meta.description}",
     "category": "{$product.category_name}",
     {if isset($product.cover)}"image" :"{$product.cover.bySize.home_default.url}",{/if}
-	{if $product.reference}"mpn": "{$product.reference}",{/if}
-    {if $product_manufacturer->name}"brand": {
+    {if $product.reference}"sku": "{$product.reference}",{/if}
+    {if $product.ean13}
+    "gtin13": "{$product.ean13}",
+    {else if $product.upc}
+    "gtin13": "0{$product.upc}",
+    {else if $product.isbn}
+    "isbn": "{$product.isbn}",
+    {else if $product.reference}
+    "mpn": "{$product.reference}",
+    {/if}    
```

Comment:
Please indent code :)
```suggestion
    {if $product.ean13}
      "gtin13": "{$product.ean13}",
    {else if $product.upc}
      "gtin13": "0{$product.upc}",
    {else if $product.isbn}
      "isbn": "{$product.isbn}",
    {else if $product.reference}
      "mpn": "{$product.reference}",
    {/if}    
```

Reply:
That will looks weird client-slide.. that's why I gave up on this

Reply:
It's JSON Ld... only robots look at this ;)